blueprint-no-op/build.gradle.kts

```diff
@@ -1,16 +1,77 @@
 @file:Suppress("UnstableApiUsage")
 
+import org.jetbrains.kotlin.gradle.ExperimentalWasmDsl
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+
 plugins {
+    alias(libs.plugins.kotlin.multiplatform)
+    alias(libs.plugins.kotlin.plugin.compose)
     alias(libs.plugins.android.library)
-    alias(libs.plugins.kotlin.android)
     alias(libs.plugins.paparazzi)
     alias(libs.plugins.binaryCompatibility)
+    alias(libs.plugins.compose)
     id("maven-publish")
 }
 
+kotlin {
+    androidTarget {
+        publishLibraryVariants("release")
+        compilations.all {
+            compileTaskProvider.configure {
+                compilerOptions {
+                    jvmTarget.set(JvmTarget.JVM_17)
+                }
+            }
+        }
+    }
+    jvm("desktop")
+    js {
+        browser()
+        useEsModules()
+    }
+    @OptIn(ExperimentalWasmDsl::class)
+    wasmJs { browser () }
```

Comment:
I think it would be great to add js demos!

But it would be probably better to do so in a separate PR. For now, it would be best to remove js support, as it is not yet tested.

As for iOS, it is a higher priority than js support, because KMP for Android/iOS code sharing is more popular right now.

If you are willing to add these demos, it would be marvelous 😄

Reply:
Okay, will try to figure out JS in a separate PR, would be really cool if it could be posted to github pages.

I've updated my branch with an iOS app; do note though that it's largely untested, the only thing I've managed to do is to run it on a simulator. Do note that I'm completely unfamiliar with iOS development and could miss some critical stuff, so please check whether this stuff works!

Reply:
Had to remove JS anyway due to issues with Jitpack anyway